src/citrine/resources/team.py

```diff
@@ -0,0 +1,291 @@
+"""Resources that represent both individual and collections of teams."""
+from typing import Optional, Union, List
+from uuid import UUID
+
+from citrine._rest.collection import Collection
+from citrine._rest.resource import Resource, ResourceTypeEnum
+from citrine._serialization import properties
+from citrine._session import Session
+from citrine._utils.functions import format_escaped_url
+from citrine.resources.user import User
+
+WRITE = "WRITE"
+READ = "READ"
+SHARE = "SHARE"
+TEAM_ACTIONS = Union[WRITE, READ, SHARE]
+
+
+class TeamMember:
+    """A Member of a Team."""
+
+    def __init__(self,
+                 *,
+                 user: User,
+                 team: 'Team',  # noqa: F821
+                 actions: TEAM_ACTIONS):
+        self.user = user
+        self.team: 'Team' = team  # noqa: F821
```

Comment:
Was flake8 just being annoying here (and on line 24)?

Reply:
There's a recursive import loop here, because a TeamMember has a Team they are part of, and a Team method `list_members` returns a TeamMember. This pattern is similar to the ProjectUser pattern we have elsewhere